ntoskrnl/ke/i386/cpu.c

```diff
@@ -416,7 +416,7 @@ KiGetFeatureBits(VOID)
                 KiCpuId(&CpuInfo, 0x80000001);
 
                 /* Check if NX-bit is supported */
-                if (CpuInfo.Edx & 0x00100000) FeatureBits |= KF_NX_BIT;
+                if (CpuInfo.Edx & X86_FEATURE_NX) FeatureBits |= KF_NX_BIT;
```

Comment:
Maybe worth putting the FeatureBits setting in its own line.

Reply:
Following existing style here, also the same Timo did on AMD64.